### PR TITLE
feat(models): created model for movie and added a serializing function

### PR DESCRIPTION
**Pull request by alikslob**
what: defines a model and a serializer
why: the data for movies needed representation 
details: 
-uses hapi bookshelf serializer 
-the model stores id, title and tear released

### CODE REVIEW

**Comment by ajpawlicki** [comment]
Usually we match the PR title to the name of the first commit. Actually, I think Github automatically does that for you. Can we keep that same standard here?

File: package.json [hunk 1]
```diff
@@ -18,7 +18,9 @@
   "scripts": {
     "lint": "eslint .",
     "start": "nodemon --ignore test/",
+
     "db:seed": "yarn db:reset && knex seed:run --knexfile db/index.js",
+
```
Comment (ajpawlicki) [inline package.json]: nit: delete newline

File: lib/models/movie.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const Bookshelf = require('../libraries/bookshelf');
+
+module.exports = Bookshelf.Model.extend({
+  tableName: 'movies',
+  serialize: function () {
```
Comment (ajpawlicki) [inline lib/models/movie.js:7]: Can we test this `serialize` function in this pr?

File: package.json [hunk 3]
```diff
@@ -18,7 +18,9 @@
   "scripts": {
     "lint": "eslint .",
     "start": "nodemon --ignore test/",
+
```
Comment (ajpawlicki) [inline package.json]: nit: delete newline

**Review by ajpawlicki — APPROVED** [review]
LGTM

**Review by tzaitsev — APPROVED** [review]
lgtm

**Pull request merged**